taskcluster/fenix_taskgraph/parameters.py

```diff
@@ -0,0 +1,55 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+import os
+import re
+
+from six import text_type
+from taskgraph.parameters import extend_parameters_schema
+from voluptuous import All, Any, Optional, Range, Required
+
+
+BETA_SEMVER = re.compile(r'^v\d+\.\d+\.\d+-beta\.\d+$')
+PRODUCTION_SEMVER = re.compile(r'^v\d+\.\d+\.\d+(-rc\.\d+)?$')
+
+
+extend_parameters_schema({
+    Required("pull_request_number"): Any(All(int, Range(min=1)), None),
+    Required("release_type"): text_type,
+    Optional("shipping_phase"): Any('build', 'ship', None),
+    Required("version"): text_type,
```

Comment:
I renamed `release_version` into `version` because that's what shipit uses.

Reply:
++